### DISCUSSION TITLE
Changing Master Node

### DISCUSSION THREAD

**Discussion by AhmedMontaaser** [Security-Onion-Solutions/securityonion#7269] [Unsupported Versions] [answered]
Hello,
Can i change master node configuration on forward node  ?

**Comment by dougburks** [comment, marked as answer]
What exactly are you trying to do?

**Reply by AhmedMontaaser** [reply]
Thanks for replying,
i am trying to extract the forward node as a OVA and deploy it with another manager with different IP and host name.
to do that i should change the configuration of forward node to connect with new master node.

**Reply by TOoSmOotH** [reply]
You would need to re-run setup. There are keys that get pushed down from the Manager that are specific to each grid deployment. Salt key, CA, SSL certs etc.

**Reply by AhmedMontaaser** [reply]
@TOoSmOotH ok thanks
